index.html

```diff
@@ -161,6 +163,31 @@ <h3 class="inner">skills</h3>
             </ul>
         </div>
     </div>
+</section>
+<section class="about-me">
+  <img src="images/Ellipse 3.svg" class="img1">
+  <img src="images/Rectangle 4.svg" class="img2">
+  <img src="images/Ellipse 12.svg" class="img3">
+  <img src="images/Ellipse 5.svg" class="img4">
+  <img src="images/Rectangle 7.svg" class="img5">
+  <img src="images/Shape.svg"class= "img6">
+<h1 class="cont">
+    Contact me
+  </h1>
+  <p class="contact-info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+  <form action="#" class="form">
```

Comment:
- [ ] Kindly make sure your `form` has a method `attribute= POST` and uses the action attribute to integrate with [formspree service](https://formspree.io/html).
- [ ] You can check [here](https://www.youtube.com/results?search_query=formspree) how to use `formspree service`